Package.swift

```diff
@@ -49,6 +49,7 @@ let package = Package(
         .package(url: "https://github.com/DiscordBM/DiscordBM.git", branch: "main"),
         .package(url: "https://github.com/DiscordBM/DiscordLogger.git", from: "1.0.0-rc.1"),
         /// Not-released area:
+        .package(url: "https://github.com/apple/swift-markdown.git", .branch("main")),
```

Comment:
Give yourself a 10-🪙 (stupid Unicode definition of "coin"...) bonus for using SwiftMarkdown ❤️❤️❤️